README.md

```diff
@@ -1,19 +1,12 @@
 # Kani Rust Verifier
 
-The Kani Rust Verifier aims to be a bit-precise model-checker for Rust.
-Kani ensures that unsafe Rust code is actually safe, and verifies that safe Rust code will not panic at runtime.
+The Kani Rust Verifier aims to be a bit-precise model checker for Rust.
 
-## Installing Kani
+Kani verifies memory safety, checks for different variants of undefined behavior, panics and user-defined assertions - even in unsafe Rust code.
 
-Until an official release is out, you can [read documentation on how to check out and build Kani yourself](https://model-checking.github.io/kani/install-guide.html).
+## Installation
 
-## What can Kani do?
-
-Our documentation covers:
-
-* [Comparison with other tools](https://model-checking.github.io/kani/tool-comparison.html)
-* [Failures that Kani can spot](https://model-checking.github.io/kani/tutorial-kinds-of-failure.html)
-* [Kani's current limitations](https://model-checking.github.io/kani/limitations.html)
+Kani must be built from source. See [the installation guide](https://model-checking.github.io/kani/install-guide.html) for more details.
```

Comment:
```suggestion
Kani currently must be built from source. See [the installation guide](https://model-checking.github.io/kani/install-guide.html) for more details.
```

Reply:
Updated as "Kani must currently be built from source" because that's what we have in the install guide. If it's not ok, let me know and I will change both.